Sparc only issue (was already in 52.x)
Crash Annotation GraphicsCriticalError: |[0][GFX1]: Unknown image format 1 (t=6.67237) [GFX1]: Unknown image format 1
Assertion failure: [GFX1]: Unknown image format 1, at /scratch/firefox/obj-sparc64-sun-solaris2.11/dist/include/mozilla/gfx/Logging.h:727
Hit MOZ_CRASH(GFX: An assert from the graphics logger) at /scratch/firefox/obj-sparc64-sun-solaris2.11/dist/include/mozilla/gfx/Logging.h:728
Couldn't convert chrome URL: chrome://branding/locale/brand.properties

00000000ffbf7dd1 libc.so.1`__lwp_sigqueue+8(b, ffbf8730, 0, a, 0, fe72eb80)
00000000ffbf7e81 libxul.so`nsProfileLock::FatalSignalHandler+0x124(b, ffbf8e10, ffbf8b00, ff00000000, 9, ffbf8730)
00000000ffbf7f41 libxul.so`js::UnixExceptionHandler+0xac(b, ffbf8e10, ffbf8b00, 0, 0, 0)
00000000ffbf7ff1 libc.so.1`__sighndlr+0xc(b, ffbf8e10, ffbf8b00, dc49a49c, 0, fe726000)
00000000ffbf80a1 libc.so.1`call_user_handler+0x354(0, ffbf8e10, 0, fe7d2a40, b, 0)
00000000ffbf8191 libc.so.1`sigacthandler+0x54(b, ffbf8e10, ffbf8b00, 0, ffbf9120, fe726000)
00000000ffbf8711 libxul.so`nsNativeThemeGTK::DrawWidgetBackground+0x1718(f089d000, d23fd380, e4818180, 56, 4f, d7514db8)
00000000ffbf8bb1 libxul.so`nsDisplayThemedBackground::PaintInternal+0xe4(d33f0120, f089d0e0, d23fd380, da699ca0, d750ac20, ffbf9460)
00000000ffbf8c71 libxul.so`nsDisplayThemedBackground::Paint+0x14(d33f0120, ffbfa648, d23fd380, feec0cd8, feec0cd8, feeb10d0)
00000000ffbf8d21 libxul.so`mozilla::FrameLayerBuilder::PaintItems+0x113c(d33c6870, d33c6780, 0, d23fd380, d33f0120, 1)
00000000ffbf9191 libxul.so`mozilla::FrameLayerBuilder::DrawPaintedLayer+0x2e0(ef109400, d23fd380, ffbf9c68, ffbf9c68, ffbf9ab8, ffbf9ca0)
00000000ffbf93a1 libxul.so`mozilla::layers::ClientPaintedLayer::PaintThebes+0x16c(ef109400, ffbf9d98, ffbf9c88, ef109400, ffbf9c68, 1)
00000000ffbf94d1 libxul.so`mozilla::layers::ClientPaintedLayer::RenderLayerWithReadback+0xd8(ef109400, ffbf9e58, 8, 0, 0, 0)
00000000ffbf95a1 libxul.so`mozilla::layers::ClientContainerLayer::RenderLayer+0x1fc(ef109000, 0, 1ce, ffbf9e58, ef109400, 0)
00000000ffbf9661 libxul.so`mozilla::layers::ClientLayerManager::EndTransactionInternal+0x1b4(e2cb8720, dacc7f10, ffbfa648, 0, ffbf9d78, ef109348)
00000000ffbf98e1 libxul.so`mozilla::layers::ClientLayerManager::EndTransaction+0x58(e2cb8720, dacc7f10, ffbfa648, 0, ffbfa020, d2318240)
00000000ffbf9991 libxul.so`nsDisplayList::PaintRoot+0x6c0(ffbfceb0, ffbfa628, ffbfa648, 0, d2318240, ef1b1800)
00000000ffbf9b11 libxul.so`nsLayoutUtils::PaintFrame+0x2420(2000, e4818020, d, ffffffffffffd6a8, ffbfa648, 184)
00000000ffbfc741 libxul.so`mozilla::PresShell::Paint+0x294(ef1d4000, ffe7e8e8, ffbfd1c8, 0, 0, e2cb8720)
00000000ffbfc911 libxul.so`nsViewManager::ProcessPendingUpdatesPaint+0x208(f26645c0, f0ef1800, ffbfd1c8, ffbfd1d0, d750ac20, f2663200)
00000000ffbfc9e1 libxul.so`nsViewManager::ProcessPendingUpdatesForView+0x404(f26645c0, f2663200, 1, 1, f0ef1800, 0)
00000000ffbfcac1 libxul.so`nsViewManager::ProcessPendingUpdates+0x44(f26645c0, 2, d75d2c90, 40, 0, 3)
00000000ffbfcb71 libxul.so`nsRefreshDriver::Tick+0x15c8(ef1b2000, 1, f26645c8, ffbfd4d0, ef1b2028, f26645c0)
00000000ffbfcf21 libxul.so`mozilla::RefreshDriverTimer::Tick+0x2a0(f08442a0, 571e299e4bca8, 53180ef77537, 1, ef1b2000, 1)
00000000ffbfcfe1 libxul.so`mozilla::VsyncRefreshDriverTimer::RefreshDriverVsyncObserver::TickRefreshDriver+0x5a8(fd6fbdf0, 531804fd2f18, b2, b2, f08442a0, 53180ef77537)
00000000ffbfd0e1 libxul.so`mozilla::VsyncRefreshDriverTimer::RefreshDriverVsyncObserver::ParentProcessVsyncNotifier::Run+0x70(d23e0580, 656e7450726f6365, 328, d23fce66, 2, ddb708a2)
00000000ffbfd191 libxul.so`nsThread::ProcessNextEvent+0xf70(fd6dc050, 2, ffbfe09f, 5a, f857a000, 0)
00000000ffbfd7e1 libxul.so`NS_ProcessNextEvent+0x24(fd6dc050, 0, 328, 0, 1b4, 0)
00000000ffbfd8a1 libxul.so`mozilla::ipc::MessagePump::Run+0x330(fd69f7c0, fe0d69f0, 147, fd69f7e0, fd6dc050, 1)
00000000ffbfd961 libxul.so`MessageLoop::RunInternal+0x60(fe0d69f0, fe0d69f0, ff000000, 1, 0, fd6dc050)
00000000ffbfda31 libxul.so`MessageLoop::Run+0x14(fe0d69f0, fe05423c, fe72eb80, fe7d2a40, fe7d2a40, ffbfe2e8)
00000000ffbfdb01 libxul.so`nsBaseAppShell::Run+0x30(fd68b9e0, 78, 328, fe726000, 114, fd6dc050)
00000000ffbfdbb1 libxul.so`nsAppStartup::Run+0xc0(fd6a7a60, 0, 328, 0, ef991090, 0)
00000000ffbfdc61 libxul.so`XREMain::XRE_mainRun+0x1d20(ffbfe990, fd6b2c10, f, ffbfe5f0, db91269c, ffbfe9c0)
00000000ffbfdfb1 libxul.so`XREMain::XRE_main+0x8e4(ffbfe990, d7a49144, fe05c270, 0, ffbfea08, 0)
00000000ffbfe0e1 libxul.so`XRE_main+0x98(1, ffbff218, ffbff080, ffbfebc8, 0, 3f)
00000000ffbfe321 libxul.so`mozilla::BootstrapImpl::XRE_main+0xc(fe0486c0, 1, ffbff218, ffbff080, fe014080, 0)
00000000ffbfe3d1 do_main+0x1ec(1, ffbff218, ffbff228, 0, ff144e38, 0)
00000000ffbfe891 main+0xa0(1, ffbff218, ffbff228, 53168285f0ca, ff, 24db98)
00000000ffbfe941 _start+0x64(0, 0, 0, 0, 0, 0)

https://bugzilla.mozilla.org/show_bug.cgi?id=1344659

diff -r 20314a5ff8d3 -r 154d39af38e7 gfx/2d/Types.h
--- a/gfx/2d/Types.h	Mon May 27 09:19:29 2019 +0200
+++ b/gfx/2d/Types.h	Mon May 27 09:25:07 2019 +0200
@@ -85,15 +85,8 @@
 // The following values are endian-independent synonyms. The _UINT32 suffix
 // indicates that the name reflects the layout when viewed as a uint32_t
 // value.
-#if MOZ_LITTLE_ENDIAN
   A8R8G8B8_UINT32 = B8G8R8A8,  // 0xAARRGGBB
   X8R8G8B8_UINT32 = B8G8R8X8   // 0x00RRGGBB
-#elif MOZ_BIG_ENDIAN
-  A8R8G8B8_UINT32 = A8R8G8B8,  // 0xAARRGGBB
-  X8R8G8B8_UINT32 = X8R8G8B8   // 0x00RRGGBB
-#else
-#  error "bad endianness"
-#endif
 };
 
 static inline int BytesPerPixel(SurfaceFormat aFormat) {
